### ISSUE TITLE
review of cross-concepts in PRGOV and PURPM

### ISSUE THREAD

**Issue by neil-ices-dk** [ices-eg/DIG#394]
we were just discussing the station use in underwater noise and it was difficult to explain the concept of Program Governance and Purpose of monitoring as it seems we are mixing concepts

PURPM https://vocab.ices.dk/?ref=1473
PRGOV https://vocab.ices.dk/?ref=42

For example, HELCOM COMBINE, MSFD or OSPAR CEMP are the **monitoring programmes** (PRGOV), and the purpose PURPM could be underwaternoise, D11 etc. At the moment we have values that confuse this:
### PRGOV: 
HELCOM Continuous Underwater Noise monitoring 
Marine Strategy Framework Directive - Descriptor 10 Litter 

So we need to review and disentangle i'm afraid.

**Comment by HansMJ** [comment]
I agree that these are not very consistent. Would be great to clean it up as it also complicates the station allocation for assessments. So note that changing PRGOV and PURPM likely requires modifications in these procedures as well

**Comment by neil-ices-dk** [comment]
for PRGOV:
<html><body>


MSFD-10 | Marine Strategy Framework Directive - Descriptor  10 Litter | PRGOV | False | 2017-04-03 | 2021-09-01
-- | -- | -- | -- | -- | --
MSFD-8 | Marine Strategy Framework Directive - Descriptor  8 Contaminants | PRGOV | False | 2021-03-18 | 2022-10-21
MSFD-9 | Marine Strategy Framework Directive - Descriptor  9 Contaminants in seafood destined for human consumption

	Marine Strategy Framework Directive - Descriptor 9 Contaminants in seafood destined for human consumption

@Osanna123 to take a look at which stations are using these labels; and cross-check that it matches PURPM. And then report back

**Comment by cekv** [comment]
Change to MSFD in PRGOV - and in the "purpose of monitoring" the descriptors (8,9,10)

**Comment by PeriklisICES** [comment]
Added PURPM to selection criteria in the station dictionary widget.

**Comment by Osanna123** [comment]
PURPM L to change the description to D10

**Comment by pcrjoana** [comment]
@Osanna123 what is the status of this, and can we close this issue?

**Comment by Osanna123** [comment]
> > PURPM L to change the description to D10
> 
> [@Osanna123](https://github.com/Osanna123) this actually came up in the EMODnet Marine Litter board of experts - do we mean for this to be used only by EU countries so EU MSFD or also by other countries that are doing policy monitoring not under the EU?

That's a very good question. Either we should create a PURPM code for each relevant policy, or make a generic one available. Normally, we'd expect a request from someone to add/review the codes.

**Comment by Osanna123** [comment]
All codes are now re-mapped, but the question is what to do with PURPM L, where MSFD D10 is not covering all litter data collections.

**Comment by neil-ices-dk** [comment]
> All codes are now re-mapped, but the question is what to do with PURPM L, where MSFD D10 is not covering all litter data collections.

could this be handled in the long description "i.e. Descriptor 10 of the MSFD, Marine Plastics EOV"

**Comment by Osanna123** [comment]
Description edited
